index.d.ts

```diff
@@ -0,0 +1,2143 @@
+
+declare namespace validator {
+    export const version: string;
+
+    /******************
+     *** Validators ***
+     ******************/
+
+/**
+* Check if the string is a valid [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) officially assigned currency code.
+*/
+    export function isISO4217(str: string): boolean;
```

Comment:
Please run a formatter like prettiert on this file, so it is formatted consitently and I don't have to comment on those parts.

Reply:
If you resolve my comments without actually changing anything, please at least tell me why.

Reply:
@rubiin can you take another look at the comments by @ST-DDT ? Also the resolved ones.
You can add index.d.ts to the list that is being linted by https://github.com/validatorjs/validator.js/blob/master/package.json#L58-L59

Reply:
sure thing will do sometime later